packages/icons/generate.ts

```diff
@@ -56,15 +56,15 @@ const generateComponentExport = (options: GenerateOptions) => {
     svgProps: {
       width: "{size}",
       height: "{size}",
-      fill: "{color}",
+      fill: "{fill}",
       "{...props}": null,
       ref: "{forwardedRef}",
     },
     plugins: sharedPlugins,
   });
   return `
 export const ${options.exportName}: IconComponent = forwardRef(
-  ({ color = "currentColor", size = 16, ...props }, forwardedRef) => {
+  ({ color = "currentColor", fill = "none", size = 16, ...props }, forwardedRef) => {
```

Comment:
What is color then?

Reply:
well, idk if worth renaming to strokeColor

Reply:
actually we don't set it much, so its easy to rename